site/src/common/blocks/RichTextBlock.tsx

```diff
@@ -141,3 +141,7 @@ const Text = styled(Typography)`
         content: " ";
     }
 `;
+
+const OrderedListItem = styled(Text)<{ depth: number }>`
```

Comment:
General question: Should we use transient props for all standard blocks since we'll upgrade to styled-components v6 with Comet v7?

Reply:
Probably a good idea 👍🏼 

```suggestion
const OrderedListItem = styled(Text)<{ $depth: number }>`
```